sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/JsonStoreDataAccess.scala

```diff
@@ -125,6 +125,19 @@ class JsonStoreDataAccess (config: CloudantConfig)  {
     val clRequest: HttpRequest = getClRequest(url, postData)
 
     val clResponse: HttpResponse[String] = clRequest.execute()
+          
+    //BAHIR-130 start
+    //In cloudant Lite number of requests pre second are limited.
+    //Just wait and try again
+    clResponse.code match {
+        case 429 => {
+            Thread sleep 1000
```

Comment:
If the server is busy, we don't want this to loop and keep trying forever.  We should have a set number of retries (our other Cloudant java library uses 3 retries) and exit with a 429 error if we reach that limit.